src/ResourceManager/Insights/Commands.Insights/Autoscale/AddAzureRmAutoscaleSettingCommand.cs

```diff
@@ -41,6 +41,7 @@ public class AddAzureRmAutoscaleSettingCommand : ManagementCmdletBase
         /// </summary>
         [Parameter(ParameterSetName = AddAzureRmAutoscaleSettingUpdateParamGroup, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The complete spec of an AutoscaleSetting")]
         [ValidateNotNullOrEmpty]
+        [Alias("InputObject")]
```

Comment:
@gucalder I would make `InputObject` the actual name of the parameter and add an alias to `SettingSpec`

Reply:
Done.